specification/nhais-adaptor.yaml

```diff
@@ -30,43 +30,50 @@ info:
     - resource names are capitalised and singular, for example the `/Patient` not `/patients`
     - array names are singular, for example `line` not `lines` for address lines
     - data items that are country-specific and thus not included in the FHIR global base resources are usually wrapped in an `extension` object
+    - data items related to an operation rather than the resource are representes as `Parameter`s
```

Comment:
spelling - representes

Reply:
thanks for the spellcheck, doesn't seem to work in the yaml file